includes/discovery/sensors/temperature/webmon.inc.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/*
+ * LibreNMS Dantel Webmon temperature sensor
+ *
+ * This program is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation, either version 3 of the License, or (at your
+ * option) any later version.  Please see LICENSE.txt at the top level of
+ * the source code distribution for details.
+ *
+ * @package    LibreNMS
+ * @link       http://librenms.org
+ * @copyright  2019 Mike Williams
+ * @author     Mike Williams <mike@mgww.net>
+ */
+
+$env = snmpwalk_group($device, 'pSlot1Table', 'WEBMON-EDGE-MATRIX-MIB');
+
+if (is_array($env)) {
```

Comment:
snmpwalk_group always returns an array.  You should check if it is empty instead.

Reply:
Fixed